ext/bin/run-load-test

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+from sys import exit, stderr, stdout
+from datetime import datetime
+import os, sys, getopt, subprocess, shutil
+
+def usage(stream):
+    usage = """
+Usage: run-load-test --host http://myhost:port [options]
+
+Options:
+  -h [--host]       Host to which the requests are made
```

Comment:
I think that we should add an option for the port, or at least inform the user that if the port is not specified, port 80 is used by default.